packages/infrastructure/README.md

```diff
@@ -0,0 +1,16 @@
+# Infrastructure
+
+## Dependencies
+
+- [azure cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
+- [terraform cli](https://learn.hashicorp.com/tutorials/terraform/install-cli)
```

Comment:
Could we add some kind of version info here so that people know what kind of minimum version they should be on. Something like:

```
terraform --version
Terraform v0.14.8
➜  ~ az --version
azure-cli                         2.20.0

core                              2.20.0
telemetry                          1.0.6

Python location '/usr/local/Cellar/azure-cli/2.20.0/libexec/bin/python'
Extensions directory '/Users/knarf/.azure/cliextensions'

Python (Darwin) 3.8.8 (default, Feb 26 2021, 23:59:43)
[Clang 12.0.0 (clang-1200.0.32.29)]

Legal docs and information: aka.ms/AzureCliLegal


Your CLI is up-to-date.

Please let us know how we are doing: https://aka.ms/azureclihats
and let us know if you're interested in trying out our newest features: https://aka.ms/CLIUXstudy
```

Reply:
Done